src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -742,9 +743,27 @@ export const ShippingRoute: FC<ShippingProps> = props => {
                 Delivery address
               </Text>
               {isAddressVerificationEnabled ? (
-                <CheckoutAddress
-                  userCountry={props.me.location?.country || "United States"}
-                  onChange={onAddressChange}
+                <AddressVerificationManager
```

Comment:
If we're to use `children` as props for `AddressVerificationManager`, then instead of passing this `render` function that returns a react component, we could just nest it:

```.jsx
<AddressVerificationManager>
	<CheckoutAddress
		userCountry={props.me.location?.country || "United States"}
		onChange={ e => {} }
		onSubmit={values => {
			verifyAddress(values).then(finalAddress => {
			onAddressChange(finalAddress)
			resetModal()
			})
		}}
	/>
</AddressVerificationManager>
```